### PR TITLE
DEV: Fix Flake8 Errors and Typos in `models/dataset/abc`

### PR DESCRIPTION
**Pull request by shaneahmed**
- Fix Flake8 Errors and Typos in `models/dataset/abc`

### CODE REVIEW

File: tiatoolbox/models/dataset/abc.py [hunk 1]
```diff
@@ -14,8 +14,8 @@ class PatchDatasetABC(ABC, torch.utils.data.Dataset):
     Attributes:
         preproc_func:
             Preprocessing function used to transform the input data. If
-            supplied, then torch.Compose will be used on the input
-            preprocs. preprocs is a list of torchvision transforms for
+            supplied, then torch. Compose will be used on the input
+            preproc. preproc is a list of torchvision or custom transforms for
```
Comment (vqdang) [inline tiatoolbox/models/dataset/abc.py:18]: no spacing for funcname, while you are at it, nest them between between `.

**Review by vqdang — APPROVED** [review]
Minor changes needed, otherwise ok

File: tiatoolbox/models/dataset/abc.py (hunk 1 above)
Comment (shaneahmed) [reply tiatoolbox/models/dataset/abc.py:18]: Do you mean to change `torch. Compose` to `torch.Compose`?

File: tiatoolbox/models/dataset/abc.py [hunk 2]
```diff
@@ -14,8 +14,8 @@ class PatchDatasetABC(ABC, torch.utils.data.Dataset):
     Attributes:
         preproc_func:
             Preprocessing function used to transform the input data. If
-            supplied, then torch.Compose will be used on the input
-            preprocs. preprocs is a list of torchvision transforms for
+            supplied, then `torch.Compose` will be used on the input
+            preproc. preproc is a list of torchvision or custom transforms for
```
Comment (vqdang) [inline tiatoolbox/models/dataset/abc.py:18]:
```suggestion
            `preproc`. `preproc` is a list of torchvision or custom transforms for
```

Comment (shaneahmed) [reply tiatoolbox/models/dataset/abc.py]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/366/commits/d6cbda02096c839860117a2aefd43b1ebee22760

File: tiatoolbox/models/dataset/abc.py [hunk 3]
```diff
@@ -14,8 +14,8 @@ class PatchDatasetABC(ABC, torch.utils.data.Dataset):
     Attributes:
         preproc_func:
             Preprocessing function used to transform the input data. If
-            supplied, then torch.Compose will be used on the input
-            preprocs. preprocs is a list of torchvision transforms for
+            supplied, then torch. Compose will be used on the input
```
[the remark this answers is not in the document]
Comment (shaneahmed) [reply tiatoolbox/models/dataset/abc.py]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/366/commits/d6cbda02096c839860117a2aefd43b1ebee22760

File: tiatoolbox/models/dataset/abc.py (hunk 1 above)
Comment (shaneahmed) [reply tiatoolbox/models/dataset/abc.py]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/366/commits/d6cbda02096c839860117a2aefd43b1ebee22760

**Pull request merged**